### ISSUE TITLE
3.0+ 版本是否支持独立分包？

### ISSUE THREAD

**Issue by lbb00** [dcloudio/uni-app#4234]
**新功能描述**

看到去年有个 pr #3260 被合并了，但 uni-app 官网的文档中没有发现相关介绍，想问下一 3.0+ 版本支持独立分包么？

**Comment by zhetengbiji** [comment]
你说的3.0是指的Vue3吗？是的话不包含，这个PR是关于Vue2的

**Comment by lbb00** [comment]
> 你说的3.0是指的Vue3吗？是的话不包含，这个PR是关于Vue2的

是的。如果 3.0 不支持的话，希望官方能跟进一下这个能力。独立分包对于业务中某些场景对首次打开速度要求比较高的，有很大的帮助。

**Comment by akFace** [comment]
请问什么时候支持独立分包？
